cmd/opampsupervisor/supervisor/supervisor_test.go

```diff
@@ -52,6 +53,7 @@ func Test_composeEffectiveConfig(t *testing.T) {
 
 	expectedConfig, err := os.ReadFile("../testdata/collector/effective_config.yaml")
 	require.NoError(t, err)
+	expectedConfig = bytes.ReplaceAll(expectedConfig, []byte("\r\n"), []byte("\n"))
```

Comment:
can this change be made in a separate PR? it seems unrelated to the change here

Reply:
No problem!

Reply:
#30690 is created @codeboten